partcad/src/partcad/project.py

```
@@ -308,17 +308,18 @@ def init_interface_by_config(self, config, source_project=None):
-            self.lock.release()
-            return p
-
         with Project.InterfaceLock(self, interface_name):
-            # Release the project lock, and continue with holding the interface lock only
-            self.lock.release()
+            # Acquire the project lock
+            self.lock.acquire()
+
+            try:
+                # See if it's already available
+                if interface_name in self.interfaces and not self.interfaces[interface_name] is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)